src/types/task.ts

```diff
@@ -10,6 +10,7 @@ export enum RequiredQueryParams {
   Execution = "execution",
 }
 
+// TODO: Remove SortBy-Limit in favor of generic TableQueryParams
```

Comment:
Is this ticketed or planned in a follow-up?

Reply:
It can be done as part of the task table work.